readme.md

```diff
@@ -1,20 +1,19 @@
 # [![AVA](media/header.png)](https://ava.li)
 
-> Futuristic test runner
-
 [![Build Status](https://travis-ci.org/avajs/ava.svg?branch=master)](https://travis-ci.org/avajs/ava)  [![Coverage Status](https://codecov.io/gh/avajs/ava/branch/master/graph/badge.svg)](https://codecov.io/gh/avajs/ava/branch/master) [![XO code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/xojs/xo) [![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/ava)
- [![Mentioned in Awesome Node.js](https://awesome.re/mentioned-badge.svg)](https://github.com/sindresorhus/awesome-nodejs)
-
-Even though JavaScript is single-threaded, IO in Node.js can happen in parallel due to its async nature. AVA takes advantage of this and runs your tests concurrently, which is especially beneficial for IO heavy tests. In addition, test files are run in parallel as separate processes, giving you even better performance and an isolated environment for each test file. [Switching](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0) from Mocha to AVA in Pageres brought the test time down from 31 to 11 seconds. Having tests run concurrently forces you to write atomic tests, meaning tests don't depend on global state or the state of other tests, which is a great thing!
-
-![](media/mini-reporter.gif)
+[![Mentioned in Awesome Node.js](https://awesome.re/mentioned-badge.svg)](https://github.com/sindresorhus/awesome-nodejs)
 
-*Read our [contributing guide](contributing.md) if you're looking to contribute (issues/PRs/etc).*
+Testing can be a slog. AVA's here to help you get it done. Its small API, detailed error output, embrace of new language features and process isolation let you write tests more effectively. So you can ship more awesome code 🚀
```

Comment:
Nitpick, but I think the sentence should still end in a `.` even though there's an emoji.
```suggestion
Testing can be a slog. AVA's here to help you get it done. Its small API, detailed error output, embrace of new language features and process isolation let you write tests more effectively. So you can ship more awesome code. 🚀
```

Reply:
I think we should also mention that it's a JavaScript test runner. Imagine someone stumbling upon this project knowing nothing about AVA.

Reply:
`small API` => `small and intuitive API` ? Or maybe: `concise API`